keystore/src/entities/platform/wasm/mls/credential.rs

```diff
@@ -38,14 +41,30 @@ impl EntityBase for MlsCredential {
         storage.get_all(Self::COLLECTION_NAME, Some(params)).await
     }
 
-    async fn save(&self, conn: &mut Self::ConnectionType) -> crate::CryptoKeystoreResult<()> {
-        let _ = self.insert(conn).await?;
-        Ok(())
+    async fn find_one(
+        conn: &mut Self::ConnectionType,
+        id: &StringEntityId,
+    ) -> crate::CryptoKeystoreResult<Option<Self>> {
+        conn.storage().get(Self::COLLECTION_NAME, id.as_slice()).await
     }
 
-    async fn insert(&self, conn: &mut Self::ConnectionType) -> CryptoKeystoreResult<Self::AutoGeneratedFields> {
-        let storage = conn.storage_mut();
+    async fn count(conn: &mut Self::ConnectionType) -> crate::CryptoKeystoreResult<usize> {
+        conn.storage().count(Self::COLLECTION_NAME).await
+    }
+}
+
+#[cfg_attr(target_family = "wasm", async_trait::async_trait(?Send))]
+#[cfg_attr(not(target_family = "wasm"), async_trait::async_trait)]
+impl EntityMlsExt for MlsCredential {
+    async fn mls_save<'a>(&'a self, storage: &WasmStorageTransaction<'a>) -> crate::CryptoKeystoreResult<()> {
+        let _ = self.mls_insert(storage).await?;
+        Ok(())
+    }
 
+    async fn mls_insert<'a>(
+        &'a self,
+        storage: &WasmStorageTransaction<'a>,
```

Comment:
Can you make this parameter name consistent with the one in the trait?

Reply:
It is auto implemented everywhere, so it is not applicable anymore